pgjdbc/src/main/java/org/postgresql/jdbc/BatchResultHandler.java

```diff
@@ -105,7 +106,10 @@ public void handleCommandStatus(String status, long updateCount, long insertOID)
     }
     latestGeneratedKeysRs = null;
 
-    longUpdateCounts[resultIndex++] = updateCount;
+    boolean canUpdateCount = !status.toUpperCase(Locale.ROOT).startsWith("SET");
```

Comment:
I would suggest avoiding `.toUpperCase()` for every execution.
WDYT of using `status.regionMatches(ignoreCase=true, ...`?

Reply:
I finally user the SqlCommandType here

Reply:
regionMatches seems to be a better choice

Reply:
Done, regionMatches is now used